contribs/gnodev/cmd/gnodev/main.go

```diff
@@ -144,6 +148,13 @@
 		"load the provided transactions file (refer to the documentation for format)",
 	)
 
+	fs.StringVar(
+		&c.genesisFile,
+		"genesis-file",
```

Comment:
What do you think about naming this `--genesis`, to keep it consistent with `gnoland`?

Maybe not even, since other files have the `-file` suffix

Reply:
good point, I like the idea to be consistant first with gnoland 👍 
And I think it's fine to only update this particular flag.

Reply:
done c0ee7d1770b528516489d67e01bc098b8e0fbc43